data/libraries.json

```
@@ -757,5 +757,14 @@
             "JavaScript"
         ],
         "url": "http://ivan.vucica.net"
+    },
+    {
+        "doap": "https://git.sr.ht/~nicoco/slidge/tree/master/item/doap.xml",
+        "last_renewed": "2023-01-18T00:00:00",
+        "name": "Slidge",
+        "platforms": [
+            "Python"
+        ],
```

```suggestion
        "platforms": null,
```

Both programming language an OS are already declared in Slidge's DOAP file. The platform key is there for legacy reasons.